tests/models/test_google.py

```diff
@@ -5020,3 +5029,58 @@ async def test_google_system_prompts_and_instructions_ordering(google_provider:
         }
     )
     assert contents == snapshot([{'role': 'user', 'parts': [{'text': 'Hello'}]}])
+
+
+async def test_google_stream_empty_chunk(
```

Comment:
What do we need this test for?

I'd like to see a test that shows that we capture the `safety_ratings` on `provider_details` and expose it in the error